lib/ace/mode/jade_highlight_rules.js

```diff
@@ -82,7 +82,7 @@ var JadeHighlightRules = function() {
         },
         {
             token : "punctuation.section.comment",
-            regex : "^\\s*\/\/(?:\\s*[^-\\s]|\\s+\\S)(?:.*$)"
+            regex : "^\\s*\/\/(?:\\s*[^\\s]|\\s+\\S)(?:.*$)"
```

Comment:
Do you know if there are some characters not allowed in jade comments?
Seems like the whole `(?:\\s*[^\\s]|\\s+\\S)` part can be removed, simply leaving `"^\\s*//.*$"`.